src/main/kotlin/apply/ui/admin/mission/JudgmentItemForm.kt

```diff
@@ -0,0 +1,43 @@
+package apply.ui.admin.mission
+
+import apply.application.EvaluationItemSelectData
+import apply.application.JudgmentItemData
+import apply.domain.mission.judgmentItem.ProgrammingLanguage
+import com.vaadin.flow.component.select.Select
+import com.vaadin.flow.component.textfield.TextField
+import support.views.BindingIdentityFormLayout
+import support.views.createItemSelect
+
+class JudgmentItemForm : BindingIdentityFormLayout<JudgmentItemData>(JudgmentItemData::class) {
+    private val programmingLanguage: Select<ProgrammingLanguage> =
+        createItemSelect<ProgrammingLanguage>("언어 선택").apply {
+            setItems(*ProgrammingLanguage.values())
+            setItemLabelGenerator { it.value }
+            value = ProgrammingLanguage.NONE
```

Comment:
c: `ProgrammingLanguage.NONE`이 필요한지 궁금합니다. 드롭다운에서 선택할 수 없도록 삭제하면 어떨까요?

```suggestion
            isEmptySelectionAllowed = false
```

Reply:
NONE은 defalut 값으로 판단하고 있기 때문에 삭제하면 안될 것 같고,
사용자 입장에서는 선택하지 못하지만 아무것도 선택하지 않으면 NONE으로 넘어오게 수정했습니다!